src/sentry/api/base.py

```diff
@@ -237,7 +237,9 @@ def paginate(
     ):
         assert (paginator and not paginator_kwargs) or (paginator_cls and paginator_kwargs)
 
-        per_page = int(request.GET.get("per_page", default_per_page))
+        per_page = int(request.GET.get('per_page') or
+                       request.GET.get('limit') or
```

Comment:
Could we change the endpoint using limit to not use limit? Standardizing the parameter sets will help down the road.

Reply:
Is `per_page` preferred?

Reply:
From a brief glance on the codebase, `limit` is used at spots where data is fetched once and pagination is not required, `per_page` is tied to a lot of default components/classes that uses pagination.

I'm leaning towards `per_page` but I'm wary of removing `limit` because it's used widely across the codebase.

Reply:
You don't have to remove `limit` but the relevant endpoint and front-end code could be adapted to use `per_page`, but fallback to `limit` if it is provided instead. This would avoid the need to update base classes which can have far reaching side-effects, and help keep `limit` for non-paginated resources.

Reply:
Sounds good. I'll change out the frontend and will not merge in this PR.